src/__tests__/actions/providers.test.js

```diff
@@ -109,7 +110,7 @@ describe('providers actions', () => {
 
       const expectedActions = [
         { type: types.OAUTH_CALLBACK_REQUEST },
-        { type: types.OAUTH_CALLBACK_SUCCESS, profileId: undefined }
+        { type: types.OAUTH_CALLBACK_SUCCESS, profileProvider: profileProvider }
```

Comment:
No need to update, just a note: ES6 lets us just use `profileProvider` in place of `profileProvider: profileProvider` if they are the same.